### ISSUE TITLE
All-Caps Hexadecimal Code Required?

### ISSUE THREAD

**Issue by reliable-justin** [FreeCodeCamp/freecodecamp#1598]
Challenge http://beta.freecodecamp.com/challenges/waypoint-use-hex-code-to-color-elements-green has an issue. Please describe how to reproduce it, and include links to screenshots if possible.

To complete this challenge the user must type her/ his hex code in all caps, as so: #00FF00. User failure to type her/ his hex code in all caps will result in her/ him being unable to move forward in the challenges.
 
Basically, it becomes very confusion (as it was for me) when the user is unable to move forward although she/ he is inputing the correct hexadecimal code. 

Reproducing this issue is simple: type the hexadecimal code in lower-caps and the challenge will be registered as incomplete. 
 
**One more note: you guys are awesome for building FCC.**

**Comment by catapixel** [comment]
```
var tests = ["assert($('body').css('background-color') === 'rgb(0, 255, 0)', 'Give your <code>body</code> element the background-color of green.')","assert(editor.match(/#00FF00/g) && editor.match(/#00FF00/g).length > 0, 'Use hex code the color green instead of the word \"green\". For example <code>body: { color: #00FF00; }</code>.')"];
```
changes to
```
var tests = ["assert($('body').css('background-color') === 'rgb(0, 255, 0)', 'Give your <code>body</code> element the background-color of green.')","assert(editor.match(/#00[Ff]{2}00/g) && editor.match(/#00[Ff]{2}00/g).length > 0, 'Use hex code the color green instead of the word \"green\". For example <code>body: { color: #00FF00; }</code>.')"];
```

**Comment by Greenheart** [comment]
@catapixel , do you want to make a pull request with your solution, or should I submit this fix? :)

**Comment by catapixel** [comment]
@greenheart if you can do it that would be great. I'm not setup to do it myself right now. Thx!

**Comment by Greenheart** [comment]
@catapixel Okay, I'll submit a pull request on this one in a moment then! :)
